tjdashboard/code/administrator/sql/install.mysql.utf8.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS `#__tj_dashboard` (
+`id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
+
+`asset_id` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+
+`ordering` INT(11)  NOT NULL ,
+`state` TINYINT(1)  NOT NULL ,
+`checked_out` INT(11)  NOT NULL ,
+`checked_out_time` DATETIME NOT NULL ,
+`created_by` INT(11)  NOT NULL ,
+`modified_by` INT(11)  NOT NULL ,
+`title` VARCHAR(255)  NOT NULL ,
+PRIMARY KEY (`id`)
+) ENGINE=MyISAM DEFAULT COLLATE=utf8_general_ci;

```

Comment:
When creating the install / update SQL, don't put the auto increment, engine and table collation. These should be picked up from database level defaults

Reply:
@irfanhanfi123  @coolbung Agree with engine and table collation but why auto increment should not be there ?